app/js/components/manageApps/ManageApps.jsx

```diff
@@ -91,7 +91,7 @@ export default class ManageApps extends React.Component {
         this.hideModal();
       }
     });
-    document.title = "Add-On Manager"
+    document.title = "Add-On Manager";
```

Comment:
Looks good to me. However you should only effect codes related to this PR else it becomes noisy. This can also be seen in other part of your code.

Reply:
I set my editor to automatically resolve any linting issues in the files I update.

Reply:
Yes